src/tools/compiletest/src/main.rs

```diff
@@ -276,6 +276,8 @@ pub fn parse_config(args: Vec<String>) -> Config {
         ),
     };
 
+    let src_base = opt_path(matches, "src-base");
+    let run_ignored = matches.opt_present("ignored");
```

Comment:
Avoiding these changes seems good for consistency's sake.

Reply:
How would you do it?

Reply:
Er, this file should be possible to leave as-is: both changes were simply extracted from struct's definition AFAICT.